src/urllib3/response.py

```diff
@@ -19,11 +19,12 @@
     ReadTimeoutError,
     ResponseNotChunked,
     IncompleteRead,
+    InvalidChunkLength,
     InvalidHeader,
     HTTPError,
 )
 from .packages.six import string_types as basestring, PY3
-from .packages.six.moves import http_client as httplib
+from .packages.six.moves import http_client as httplib  # noqa: F401
```

Comment:
Why do we need the import? I think you removed the only use of `httplib` here

Reply:
There is a use of ```urllib3.response.httplib``` in two places in the tests:
https://github.com/urllib3/urllib3/blob/86fb90fdafee0077f4b90cbb2b9c3be99d4a1991/test/test_connectionpool.py#L13
https://github.com/urllib3/urllib3/blob/86fb90fdafee0077f4b90cbb2b9c3be99d4a1991/test/with_dummyserver/test_socketlevel.py#L12

I guess there is no reasonable excuse to export the import from ```test/response.py```, do you think moving it to ```test/__init__.py``` a more appropriate solution?

Reply:
Maybe we should just use the six import in tests? In any case, we should stop importing httplib3 from urllib3.response, yeah.

Reply:
Would you like to do this here or in a another pull request?